common/app/views/support/Commercial.scala

```diff
@@ -34,9 +34,10 @@ object Commercial {
     }
 
     def adSizes(metaData: MetaData, edition: Edition): Map[String, Seq[String]] = {
+      val fabricAdverts = if (FabricAdverts.isSwitchedOn) Some("88,71") else None
```

Comment:
I actually prefer this, because it's symbolic, rather than the strings below, like "88,70". Not sure why we don't use a pre-defined key value to do the targeting, @kenlim ?